src/gui/PhysicalGameStatePanel.java

```diff
@@ -252,9 +291,9 @@ public static void draw(Graphics2D g2d,
         int grid = Math.min(gridx,gridy);
         int sizex = grid*pgs.getWidth();
         int sizey = grid*pgs.getHeight();
-        int unitLineThickness = 1;
-        if (grid > 10) unitLineThickness = 2;
-        if (grid > 20) unitLineThickness = 4;
+        float unitLineThickness = 1f;
+        if (grid > 10) unitLineThickness = 1.8f;
+        if (grid > 20) unitLineThickness = 2.3f;
```

Comment:
oh, was this too thick? I made it that thick as people complained that for screenshots, it was hard to see the outline in small maps :)